lib/BuildSystem/ExternalCommand.cpp

```diff
@@ -322,7 +322,7 @@ ExternalCommand::computeCommandResult(BuildSystemCommandInterface& bsci) {
       // info, but need to refactor the command result to just store the node
       // subvalues instead.
       FileInfo info{};
-      info.size = bsci.getBuildEngine().getCurrentTimestamp();
+      info.size = bsci.getBuildEngine().getCurrentIteration();
```

Comment:
I suggest we rename to "Epoch", which often has the connotation we want.

Reply:
Done and also separated into an individual commit 👍